book/src/SUMMARY.md

```diff
@@ -1,11 +1,30 @@
 # Table of contents
 
-* [The Solana Codebase](introduction.md)
-* [Building from Source](getting-started/README.md)
-  * [Example Client: Web Wallet](getting-started/webwallet.md)
-* [Programming Model](programs/README.md)
-  * [Example: Tic-Tac-Toe](programs/tictactoe.md)
-  * [Drones](programs/drones.md)
+* [Introduction](introduction.md)
+* [Using Solana](using-solana.md)
+  * [Command-line Client](cli.md)
+  * [Paper Wallet](paper-wallet/README.md)
+    * [Installation](paper-wallet/installation.md)
+    * [Paper Wallet Usage](paper-wallet/usage.md)
+  * [Offline Signing](offline-signing/README.md)
+    * [Durable Transaction Nonces](offline-signing/durable-nonce.md)
+* [Developing Applications](apps/README.md)
+  * [Example: Web Wallet](apps/webwallet.md)
+  * [Example: Tic-Tac-Toe](apps/tictactoe.md)
+  * [Drones](apps/drones.md)
+  * [Anatomy of a Transaction](transaction.md)
+  * [JSON RPC API](apps/jsonrpc-api.md)
+  * [JavaScript API](apps/javascript-api.md)
```

Comment:
Big fan of this grouping for developers. But I think "JSON RPC API" section is also relevant to validators. Maybe we could add a link to it in the "Running a Validator" section as well?

Reply:
To my knowledge, validators primarily use the CLI and would only use those curl commands if the CLI was missing the command.

Reply:
Ah true, I've observed that as well. Never mind!